topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -871,10 +877,224 @@ These datasets can be used for the downstream analysis, e.g. mapping.
 > {: .solution}
 {: .question}
 
+# Assess quality with Nanoplot - Long reads only
+
+In case of long reads, we can check sequence quality with [Nanoplot](https://github.com/wdecoster/NanoPlot/) {% cite 10.1093/bioinformatics/bty149 %}. It provides basic statistics with nice plots for a fast quality control overview.
+
+> ### {% icon hands_on %} Hands-on: Quality check of long reads
+> 1. Create a new history for this part and give it a proper name
+>
+> 2. Import the PacBio HiFi reads `m64011_190830_220126.Q20.subsample.fastq.gz` from [Zenodo](https://zenodo.org/record/5730295)
+>
+>    ```
+>    https://zenodo.org/api/files/ff9aa6e3-3d69-451f-9798-7ea69b475989/m64011_190830_220126.Q20.subsample.fastq.gz
+>    ```
+>
+> 3. {% tool [Nanoplot](toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot/1.28.2+galaxy1) %} with the following parameters
+>    - {% icon param-files %} *"files"*: `m64011_190830_220126.Q20.subsample.fastq.gz`
+>    - *"Options for filtering or transforming input prior to plotting"*
+>        - {% icon param-select %} *"Specify the bivariate format of the plots."*: `dot`, `kde`
+>        - {% icon param-select %} *"Show the N50 mark in the read length histogram."*: `Yes`
+>
+> 4. Inspect the generated HTML file
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> What is the mean Qscore ?
```

Comment:
this is a new way of doing solution boxes! after N years still people find new ways of doing things :laughing:  You're welcome to keep it this way but most people just make the questions a numbered list, and the same for the solutions, thus only one solutions block.

Reply:
Oops 😅 It's my first GTN so I am still learning :)